sdk/core/core-tracing/lib/interfaces/plugin.ts

```diff
@@ -0,0 +1,10 @@
+import { Tracer } from "./tracer";
```

Comment:
This is a file we added (because we've extended Tracer to have this pluginType member.) It's not from the OpenTelemetry repo.

Also I think I can deprecate/remove this in the next PR. :)

Reply:
We should also have copyright headers for the files we author. I think that was missed when this package was created, but in most of our other packages our files have a copyright.

(I heard we have a linter that will even add it, but not sure if it's configured for this package)